aa_stripe/models.py

```diff
@@ -401,6 +401,7 @@ def charge(self):
         self.customer = customer
         if customer:
             metadata = {"object_id": self.object_id, "content_type_id": self.content_type_id}
+            idempotency_key = "{}-{}".format(metadata["object_id"], metadata["content_type_id"])
```

Comment:
This is wrong because dunning flow is generating more than 1 charge objects for the same order.

Reply:
@remik Do we have any kind of dunning id generated?

Reply:
you can try charge attempt number but it is Ro specific so here need to be universal.